src/main/java/org/apache/brooklyn/cloudfoundry/entity/VanillaPaasApplicationCloudFoundryDriver.java

```diff
@@ -51,6 +50,7 @@
     private VanillaCloudFoundryApplicationImpl entity;
     private String applicationName;
     private String applicationUrl;
+    protected String localArtifactPath;
```

Comment:
It is necessary to maintain the `localArtifactPath` because it is needed to update the resources profile due to [cf-java-client#issue544](https://github.com/cloudfoundry/cf-java-client/issues/544)